pom.xml

```diff
@@ -39,11 +43,263 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.7.0</version>
 				<configuration>
-					<source>1.8</source>
-					<target>1.8</target>
+					<source>${maven.compiler.release}</source>
+					<target>${maven.compiler.release}</target>
```

Comment:
You shouln'd need `<source>` and `<target>` anymore when using `maven.compiler.release`